docs/content/instrumenting/extra-options.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Extra Options
+weight: 7
+---
+
+# Labels
```

Comment:
It's probably worth having Labels as a separate item since they are a core part of using this library.

Reply:
Moved labels and exemplars to their own section. `## Disabling `_created` metrics` is now in the `_index.md` of instrumenting. I can change if need it.